AdminWebsite/AdminWebsite/Controllers/ReferenceDataController.cs

```diff
@@ -49,7 +49,6 @@ public async Task<ActionResult<IList<HearingTypeResponse>>> GetHearingTypes()
             var reult = caseTypes.SelectMany(caseType => caseType.HearingTypes.Select(hearingType => new HearingTypeResponse
             {
                 Group = caseType.Name,
-                Code = string.Empty, // not used anymore
```

Comment:
This was removed to make tests pass due to the bookings api client having been updated, which returns null for the Code